lib/ansible/modules/system/lvg.py

```diff
@@ -35,7 +35,7 @@
     type: list
   pesize:
     description:
-    - The size of the physical extent. pesize must be a power of 2, or multiple of 128KiB.
+    - The size of the physical extent. pesize must be a power of 2.
```

Comment:
Isn't this change wrong? Values not equal to a power of two are explicitly allowed! The manpage says:

> The value must be either a power of 2 of at least 1 sector (where the sector size is the largest sector size of the PVs currently used in the VG), **or at least 128KiB.**

(emphasis is mine)

Reply:
sorry about that. Thought I understood the issue and the PR. I'll figure out how to revert this (since the github revert button isn't working for me )

Reply:
No problem. I just noticed this PR when #29295 was closed; I didn't notice the notification there from this PR... (Too bad github isn't sending notification emails when a PR/issue is referenced...)